src/foam/u2/crunch/CapabilityFeatureView.js

```diff
@@ -110,18 +112,21 @@ foam.CLASS({
             'background-image': "url('" + self.data.icon + "')"
           })
           .add(this.slot(function(cjStatus) {
-            return this.E().addClass(style.myClass('tooltip'))
+            return this.E()
+            .addClass(style.myClass('tooltip'))
               .start('span')
                 .addClass(style.myClass('tooltiptext'))
                 .addClass(style.myClass('tooltip-bottom'))
                 .enableClass(style.myClass('tooltipDisabled'), self.tooltipEnabled, true)
                 .add(cjStatus.documentation)
               .end()
-              .add(cjStatus.label).addClass(style.myClass('badge'))
-              .style({
-                'background-color': cjStatus.background,
-                'color': cjStatus.color
-              });
+              .start(foam.u2.view.BadgeView, {label: cjStatus.label, color: cjStatus.color, backgroundColor: cjStatus.background, isAttachedToCard: true}).addClass(self.myClass('badge')).end()
+              // .add(cjStatus.label).addClass(style.myClass('badge'))
+              // .style({
```

Comment:
If this is not going to be used any more, we should be deleting it. There will always be a history of it in github if you ever want to come back.

Reply:
Missed this somehow, will delete